pkg/util/sys/device.go

```diff
@@ -280,25 +279,12 @@ func GetDiskUUID(device string, executor exec.Executor) (string, error) {
 	return parseUUID(device, output)
 }
 
-func GetPartitionLabel(deviceName string, executor exec.Executor) (string, error) {
-	// look up the partition's label with blkid because lsblk relies on udev which is
-	// not available in containers
-	devicePath := fmt.Sprintf("/dev/%s", deviceName)
-	cmd := fmt.Sprintf("udevadm %s", devicePath)
-	output, err := executor.ExecuteCommandWithOutput(false, cmd,
-		"udevadm", "info", "--query=property", devicePath)
-	if err != nil {
-		return "", fmt.Errorf("failed to get partition label for device %s: %+v", deviceName, err)
-	}
-
-	return parsePartLabel(output), nil
-}
-
+// MountDevice mount a device
 func MountDevice(devicePath, mountPath string, executor exec.Executor) error {
 	return MountDeviceWithOptions(devicePath, mountPath, "", "", executor)
 }
 
-// comma-separated list of mount options passed directly to mount command
+// MountDeviceWithOptions comma-separated list of mount options passed directly to mount command
```

Comment:
Add more detail than just the name?
```suggestion
// MountDeviceWithOptions mounts a device with a comma-separated list of mount options passed directly to  'mount' command
```

Reply:
This function is gone now :)